### PR TITLE
nxos_install_os DOCSTRING

### PR DESCRIPTION
**Pull request by GGabriele**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

nxos_install_os
##### ANSIBLE VERSION

```
ansible 2.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### SUMMARY

Adding more details on DOCSTRING about how to use the module.

### CODE REVIEW

File: network/nxos/nxos_install_os.py [hunk 1]
```diff
@@ -27,7 +27,10 @@
     - The module will fail due to timeout issues, but the install will go on
       anyway. Ansible's block and rescue can be leveraged to handle this kind
       of failure and check actual module results. See EXAMPLE for more about
-      this.
+      this. The first task on the rescue block is needed to make sure the
+      device has completed all checks and it started to reboot. The second
+      task is needed to wait the device to come back up. Last two tasks are
```
Comment (jjshoe) [inline network/nxos/nxos_install_os.py:32]: "need to wait FOR the device" ?

Comment (jjshoe) [reply network/nxos/nxos_install_os.py:32]: "THE last two tasks"?

File: network/nxos/nxos_install_os.py [hunk 2]
```diff
@@ -27,7 +27,10 @@
...
+      used to verify the installation process's been successful.
```
Comment (jjshoe) [inline network/nxos/nxos_install_os.py:33]: "installation process was successful"

**Comment by GGabriele** [comment]
#5130

**Pull request merged**